### ISSUE TITLE
Spotify Player not opening when searching for a song

### ISSUE THREAD

**Issue by Luismi74** [raycast/extensions#9692]
### Extension

https://raycast.com/mattisssa/spotify-player

#### Error:
```
Error: No active device found. Spotify is not installed.
    at Bn (/Users/luis/.config/raycast/extensions/320f40ef-a633-415a-ab0e-1e99515478f7/search.js:59:1352)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async n (/Users/luis/.config/raycast/extensions/320f40ef-a633-415a-ab0e-1e99515478f7/search.js:61:493)
```

### Steps To Reproduce

1. Open Raycast
2. Opened Search Spotify function from the extensions
3. Typed a song and hit enter to play it

### Current Behaviour

Weird behavior since I do have Spotify installed. Not opening at all.

### Expected Behaviour

Open and play the song requested

**Comment by jatindotdev** [comment]
duplicate of #9649, will be fixed in #9682

**Comment by jatindotdev** [comment]
@raycastbot close this issue
